src/main/java/io/pinecone/PineconeControlPlaneClient.java

```diff
@@ -29,17 +27,17 @@ public PineconeControlPlaneClient(String apiKey, OkHttpClient okHttpClient) {
         manageIndexesApi.setApiClient(apiClient);
     }
 
-    public IndexModel createIndex(CreateIndexRequest createIndexRequest) {
+    public IndexModel createIndex(CreateIndexRequest createIndexRequest) throws PineconeException {
```

Comment:
These should all throw `PineconeException` from the catch blocks when calling `handleApiException`.

Reply:
This will improve readability!